assets/src/js/Components/ArchivePicker/ArchivePicker.js

```diff
@@ -185,7 +215,7 @@ const reducer = (state, action) => {
   }
 };
 
-export default function ArchivePicker() {
+export default function ArchivePicker({mediaView, adminView}) {
```

Comment:
What do you think if we move these two props into the context?

Reply:
In addition to that, instead of using 2 different props, we can just use one like `view` which accepts `media|admin`

Reply:
@Osong-Michael, have you checked this?
```javascript
export const MEDIA = 'media';
export const ADMIN = 'admin

export default function ArchivePicker({view}) {
...
}
...
{(image && view === ADMIN) && (
```
Less props we pass, easier is to scale it.

Reply:
Oh I see what you are saying, in that case we don't need the extra variables then, we could just use the `view` prop and check if it is _'admin'_ or _'editor'_ and call the component like `< ArchivePicker view="admin" />`.  What do you think @GP-Dan-Tovbein ?

Reply:
One more thing.. you can store the `view` prop to the `reducer` and then is shared to all nested components (which uses the `Context`). And finally, you'll avoid to pass the value like you do [here](https://github.com/greenpeace/planet4-master-theme/pull/2112/files#diff-2d5f38a9e36d2b751a99282786e7adfec59d239898030fce76b89ddbbeb6a426R318), but pass as a dependency.

For instance,
Instead of 
```javascript
const includeInWp = async (ids = [], isMediaView = false) => { ... }
```
Do this
```javascript
const includeInWp = useCallback(async (ids = []) => {}, [view]);
```